eclass/llvm-utils.eclass

```diff
@@ -132,7 +132,7 @@ llvm_prepend_path() {
 	# prepend new path in front of the first LLVM version found
 	for x in "${split_path[@]}"; do
 		if [[ ${x} == */usr/lib/llvm/*/bin ]]; then
-			if [[ ${x} != ${llvm_path} ]]; then
+			if [[ ${x} != ${llvm_path} && ! ${added} ]]; then
```

Comment:
That should work, although I find my other suggestion easier to understand.

I really wanted to comment on the commit message:
> Thanks to @miller-alex who spotted it in:
#35196 (comment)

I never use that github account name anywhere, I'd prefer `Alexander Miller <alex.miller@gmx.de>` (or just the name) like all my other contributions.

Reply:
I think your suggestion actually could result in duplicates if first entry in the path matches the selected path. But yeah, the logic is kinda ugly, I need to take a second look at it.

Reply:
Actually, you were right. I've added some more test cases, and actually arrived at the same code. Thanks again!